aspnet/mvc/models/validation.rst

```diff
@@ -1,8 +1,125 @@
-.. include:: /../common/stub-topic.txt
+Model Validation 
+================
+By `Rachel Appel <http://github.com/rachelappel>`_
 
-|stub-icon| Model Validation
-============================
+In this article:
 
-.. include:: /../common/stub-notice.txt
+- `Introduction to model validation`_
+- `Data annotations`_
+- `Model state`_
+- `Custom validation`_
+- `Client side validation`_ 
+- `Remote valiation`_
 
-.. _issue: https://github.com/aspnet/Docs/issues/116
+Introduction to model validation
+--------------------------------
+Before an app stores data in a database, the app must validate the data. Data must be scrubbed for security, verified that it is appropriately formatted by type and size, and it must conform to your rules. Validation is necessary although it can be redundant and tedious to implement. In MVC, validation happens on both the client and server. 
+
+By using data annotations in MVC apps you can validate data by customizing models so they mimic the underlying structure of the data store. This way you can enforce integrity rules and constraints against models by applying attributes to model properties. These attributes generate validation code, thereby reducing the amount of code you must write. Fortunately, .NET has abstracted validation into the data annotations, often reducing validation to a single line of code.
```

Comment:
- Suggest `By using` -> `With`
- Suggest removing `so they mimic the underlying structure of the data store. This way` and removing mention of `integrity rules`.